startup/10-motors.py

```diff
@@ -96,9 +97,25 @@ class Blades(Device):
 elif beamline_stage == 'open_MAXS':
     smx = EpicsMotor('XF:11BMB-ES{Chm:Smpl2-Ax:X}Mtr', name='smx')
     smy = EpicsMotor('XF:11BMB-ES{Chm:Smpl2-Ax:Y}Mtr', name='smy')
+    #sth = EpicsMotor('XF:11BMB-ES{SM:2-Ax:theta}Mtr', name='sth')
+    #schi = EpicsMotor('XF:11BMB-ES{SM:2-Ax:chi}Mtr', name='schi')
```

Comment:
I think we should use version control to manage comments like this.

Reply:
Same for other places where comments are used to track old code.
If the comments are used for switching between code, then a branch might be a good idea.

Reply:
agreed!

Reply:
This should also be turned into an actualy stage Device rather than a bunch of loose motors.